libbeat/scripts/Makefile

```diff
@@ -206,18 +206,6 @@ coverage-report:
 update:
 	bash ${ES_BEATS}/libbeat/scripts/update.sh ${BEATNAME} ${BEAT_DIR}/${BEATNAME} ${ES_BEATS}/libbeat
 
-### KIBANA FILES HANDLING ###
-
-KIBANA_ES?=http://localhost:9200/
-.PHONY: update
-kibana-export:
```

Comment:
Yes, I will them back in the Makefile once I check the add details about how to use the new script.